log.go

```diff
@@ -1,7 +1,11 @@
 package work
 
-import "fmt"
+import (
+	"fmt"
+	"runtime/debug"
+)
 
 func logError(key string, err error) {
 	fmt.Printf("ERROR: %s - %s\n", key, err.Error())
+	debug.PrintStack()
```

Comment:
I don't think this is doing what you think it is. Isn't this just printing the stacktrace at the time of the call and will always print the same exact stack trace regardless of the error?

The canonical "Go way" of dealing with errors and stacktraces is: https://godoc.org/github.com/pkg/errors

Refactoring gocraft/work to use pkg/errors is a little more of an undertaking, but likely worth the effort.